docs/examples/driver_examples/QCoDeS example with Galil DMC4133 Controller.ipynb

```diff
@@ -0,0 +1,895 @@
+   "metadata": {},
+   "source": [
+    "# Calibration"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8d66d4b5",
+   "metadata": {},
+   "source": [
+    "Check the state of the motors."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 62,
+   "id": "2442e7a1",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-09-06T12:46:08.312660Z",
+     "start_time": "2021-09-06T12:46:08.282613Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'A': -90000, 'B': -10000, 'C': -16000}\n",
+      "on\n",
+      "on\n",
+      "on\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(controller.absolute_position())\n",
+    "print(controller.motor_a.on_off_status())\n",
+    "print(controller.motor_b.on_off_status())\n",
+    "print(controller.motor_c.on_off_status())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "c5aade6d",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-09-06T12:19:49.659764Z",
+     "start_time": "2021-09-06T12:19:49.649603Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "controller.motors_off()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "e7e2eb5b",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-09-06T12:19:53.183446Z",
+     "start_time": "2021-09-06T12:19:53.163445Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "arm.set_arm_kinematics()  # sets default values of arm speed to be 100 micro meters per second,\n",
+    "                          # acceleration and deceleration to be 2048 micro meters per second square"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d73781eb",
+   "metadata": {},
+   "source": [
+    "Manually move the motors and take the needle head to the position where you want to set the origin. This will be the left bottom corner of the chip and run following commands."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "1c0ac718",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-09-06T12:20:57.094423Z",
+     "start_time": "2021-09-06T12:20:57.084417Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "controller.define_position_as_origin()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 54,
+   "id": "a6389d1d",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-09-06T12:43:16.312056Z",
+     "start_time": "2021-09-06T12:43:16.302053Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "arm.set_left_bottom_position()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9c1b17cc",
+   "metadata": {},
+   "source": [
+    "From now on all the motor movements will be controlled by the driver commands.\n",
+    "\n",
+    "Next step is to set reverse limits for for all three motors. First take the motor C to the extreme reverse position which you want to set as the reverse limit with following command."
```

Comment:
what about forward limits?

Reply:
i see, makes then. then i'd explain this in the notebook (e.g. add this exact comment to the notebook). and even despite the lack of sense users still want to be extra careful and set the forward limits, they still can, right?

Reply:
Ok, I will add this info to the notebook and yes, forward limits can be set.

Reply:
Updated as discussed